cpp/unreal_plugins/SimulationController/Source/SimulationController/UrdfBotAgent.h

```diff
@@ -20,7 +17,7 @@ class UWorld;
 class AUrdfBotPawn;
 class CameraSensor;
 class ImuSensor;
-class SonarSensor;
+//class SonarSensor;
```

Comment:
We decided recently to remove SonarSensor entirely until we have a chance to clean it up and generalize it into a general-purpose raycasting sensor.